src/widget/weffectchainpresetbutton.cpp

```diff
@@ -46,20 +46,23 @@ void WEffectChainPresetButton::populateMenu() {
     m_pMenu->clear();
 
     // Chain preset items
-    bool chainIsPreset = false;
+    bool allowUpdatingPreset = false;
```

Comment:
Can we invert the logic her to match the function call?

Reply:
Like this?
```
bool isReadOnly = true;
...
isReadOnly = pChainPreset->isReadOnly();
...
if (!isReadOnly) {
   // show Update button
```

Reply:
Yes

Reply:
alright, done.